source/adl-standalone/tests/aggregate.yml

```diff
@@ -0,0 +1,292 @@
+database_name: "test"
+
+collection_name: &collection_name 'quux'
+
+tests:
```

Comment:
Similarly, should we be testing all options for all of these commands, e.g. for aggregate:

* batchSize
* maxTimeMS
* collation
* comment

And similarly for all of the other commands.

Reply:
It's not.  But the scope just says: "and query".  So there is a lot of latitude. @craiggwilso, are you comfortable with the level of coverage we have now?